Uplift/Views/Cells/FacilitiesDropdownCell.swift

```diff
@@ -0,0 +1,185 @@
+//
+//  FacilitiesDropdownCollectionViewCell.swift
+//  
+//
+//  Created by Cameron Hamidi on 11/6/19.
+//
+
+import UIKit
+
+class FacilitiesDropdownCell: UICollectionViewCell {
+
+    private enum FacilityName: String {
+        case bowlingLanes = "BOWLING LANES"
+        case equipment = "EQUIPMENT"
+        case fitnessCenter = "FITNESS CENTER"
+        case gymnasium = "GYNMASIUM"
+        case swimmingPool = "SWIMING POOL"
+    }
+
+    private let headerImageView = UIImageView()
+    private let headerNameLabel = UILabel()
+    private let headerOpenLabel = UILabel()
+    private let headerViewHeight: CGFloat = 52
+    private let minimumInteritemSpacing: CGFloat = 16
+
+    private var collectionView: UICollectionView!
+    private var collectionViewHeight: CGFloat = 0
+    private var dropdownView: DropdownView!
+    private var facility: Facility!
+    private var headerView: UIView!
```

Comment:
make `private let`. i also don't think you currently initialize this to any value?

Reply:
bump